### PR TITLE
add flag to encrypt_util for length delimited binary format

### PR DESCRIPTION
**Pull request by eriktaubeneck**
In order to support the new ability for helpers to load data from a file/url, we need the ability to produce length delimited encrypted data for testing. This adds a flag to the `crypto_util` called `--length-delimited` which will produce length delimited binary instead of newline delimited hex.

The next step is to use this with the integration test. The flow will be

1. generate raw input data files (one for each shard)
2. use `crypto_util` to encrypt each of these for each shard (and each helper)
3. pass these files into the report collector (is this implemented yet @akoshelev ?)
4. cat all input data files together and run `in-the-clear`
5. compare results

### CODE REVIEW

File: ipa-core/src/cli/crypto/hybrid_encrypt.rs
Comment (andyleiserson) [inline ipa-core/src/cli/crypto/hybrid_encrypt.rs:57]: Instead of a boolean argument it might be clearer to have descriptive options (something like the `FileFormat` choices -- possibly clap can even be taught to parse into `FileFormat`).

**Review by andyleiserson — APPROVED** [review]

File: ipa-core/src/cli/crypto/hybrid_encrypt.rs
Comment (eriktaubeneck) [reply ipa-core/src/cli/crypto/hybrid_encrypt.rs:57]: I generally think of [docopt](http://docopt.org) as the canonical way to design CLIs, and this would probably fit as a mutually exclusive element better than an element with an arg. Clap probably has a way of defining mutually exclusive elements (and as you say, clap could likely be taught to coerce that into the enum.) This would be even more important if there were more than two options. 

I likely won't get to this before I'm out, so up to @andyleiserson and @akoshelev if you want to refactor before merging.

**Review by akoshelev — APPROVED** [review]

**Pull request merged**
